e2e/specs/administration/predefinedResolutions.spec.ts

```diff
@@ -0,0 +1,182 @@
+
+            it('I can Update Predefined Resolution', async () => {
+                const updateMessage = 'Predefined Resolution was updated!';
+                await predefinedResolutions.updateResolution(edit.resolution.name, predefinedResolutions.columns.resolution,
+                    create.resolution.name, predefinedResolutions.columns.resolution);
+                await predefinedResolutions.notification.assertIsSuccess(updateMessage);
+
+                await predefinedResolutions.updateResolution(edit.comment, predefinedResolutions.columns.comment,
+                    edit.resolution.name, predefinedResolutions.columns.resolution);
+                await predefinedResolutions.notification.assertIsSuccess(updateMessage);
+
+                await predefinedResolutions.updateResolution(edit.expression, predefinedResolutions.columns.expression,
+                    edit.resolution.name, predefinedResolutions.columns.resolution);
+                await predefinedResolutions.notification.assertIsSuccess(updateMessage);
+
+                await predefinedResolutions.updateResolution(usersTestData.localAdmin.user_name, predefinedResolutions.columns.assignee,
+                    edit.resolution.name, predefinedResolutions.columns.resolution);
+                return predefinedResolutions.notification.assertIsSuccess(updateMessage);
+            });
+
+            it('I can see that import affected by Predefined Resolution', async () => {
+                const imported: TestRun = (await projectHelper.importer
+                    .executeCucumberImport('Predefined Resolution', [cucumberImport], ['cucumber.json']))[0];
+                const results: TestResult[] = await projectHelper.editorAPI.getResults({
+                    test_run_id: imported.id,
+                    project_id: projectHelper.project.id
+                });
+
+                const result = results.find(x => x.fail_reason === 'step was skippedstep was skipped');
+                if (result) {
+                    expect(result.assigned_user.user.user_name).toBe(usersTestData.localAdmin.user_name, 'Assignee was not filled!');
+                    expect(result.test_resolution.name).toBe(edit.resolution.name, 'Resolution was not filled!');
+                    expect(result.comment).toBe(edit.comment, 'Comment was not filled!');
+                } else {
+                    expect(result).toBeDefined('The imported results do not contain expected result!');
+                }
+            });
+
+            it('I can remove Predefined Resolution', async () => {
+                await predefinedResolutions.clickRemoveResolution(edit.expression);
+                await expect(predefinedResolutions.modal.isPresent())
+                    .toBe(true, 'Confirmation modal is not shown!');
+                await predefinedResolutions.modal.clickYes();
+                return predefinedResolutions.notification.assertIsSuccess('Predefined Resolution was removed!');
```

Comment:
should we check that resolution is absent in table? 
now, you check only alert

Reply:
fixed